ui/app/AppLayouts/Wallet/panels/WalletFooter.qml

```diff
@@ -28,6 +28,11 @@ Rectangle {
 
     color: Theme.palette.statusAppLayout.rightPanelBackgroundColor
 
+    QtObject {
+        id: d
+        readonly property int collectibleTransferableTrait: !!walletStore.currentViewedCollectible && sendButton.visible ? walletStore.findCollectibleTrait(walletStore.currentViewedCollectible, Constants.collectibleTrait.transferable, Constants.collectibleTrait.transferableNoValue) : -1
```

Comment:
hmm we shouldn't really be using these traits for anything other than display purposes... Anybody could add a trait called `Transferable` to their NFT but still allow people to send it.
IMO this should be a separate field coming from the backend